src/main/java/com/TableTalk/Enterprise/controllers/TableTalkController.java

```diff
@@ -1,14 +1,8 @@
 package com.TableTalk.Enterprise.controllers;
 
-import com.TableTalk.Enterprise.dto.Game;
-import com.TableTalk.Enterprise.dto.ProfilePicture;
-import com.TableTalk.Enterprise.dto.Room;
-import com.TableTalk.Enterprise.dto.User;
+import com.TableTalk.Enterprise.dto.*;
```

Comment:
Just import it all if you need everything

Reply:
This is not recommended, for documentation purposes.  With an *, you don't know what classes are imported from the package.  This poses an additional problem if you have duplicate class names, like java.sql.Date and java.util.Date.  I'd leave this as it were.